src/mainwindow.cpp

```diff
@@ -63,6 +63,15 @@ MainWindow::MainWindow(QWidget *parent) :
             mp_ui->mainToolBar, &TopWidget::updateBackForwardButtons);
     connect(mp_ui->tabBar, &TabBar::libraryPageDisplayed,
             this, &MainWindow::when_libraryPageDisplayed);
+
+    connect(mp_ui->tabBar, &TabBar::currentTitleChanged,
+            &(mp_ui->mainToolBar->getSearchBar()), &SearchBar::on_currentTitleChanged);
+
+    // This signal emited more often than the history really updated
+    // but for now we have no better signal for it.
+    connect(mp_ui->tabBar, &TabBar::currentTitleChanged,
+            mp_ui->mainToolBar, &TopWidget::updateBackForwardButtons);
```

Comment:
Several lines above there is

```
    connect(mp_ui->tabBar, &QTabBar::currentChanged,
    mp_ui->mainToolBar, &TopWidget::updateBackForwardButtons);
```

Are they both justified? If yes, shouldn't `SearchBar::on_currentTitleChanged` be connected to `QTabBar::currentChanged`, too?

Reply:
> Are they both justified? If yes, shouldn't `SearchBar::on_currentTitleChanged` be connected to `QTabBar::currentChanged`, too?

SearchBar::on_currentTitleChanged connection mistakenly fell in the other commit, fixed.